docs/pico_sdk.md

```diff
@@ -6,19 +6,22 @@ Follow the Raspberry Pi Foundation's
 setup a proper development environment on your host PC (the machine that
 will build your project).
 
-It is important to remember that the pico-sdk repository should exist in the same directory that the RF24 repository (or your project's directory) exists.
+Either set an environment variable named `PICO_SDK_PATH` that points to your
+local clone of the pico-sdk or put the pico-sdk next to the `RF24` folder or
+the folder containing your project using the `RF24` library:
```

Comment:
I'm ok with RF24 being in inline code blocks, but I just wanted to mention that doxygen automatically adjusts all instances of the word "RF24" as a hyperlink to the `RF24` class' docs.

Reply:
Oh, missed that when doing the corrections. Please feel free to change to quotation marks.